iidm/iidm-extensions/src/main/java/com/powsybl/iidm/network/extensions/VoltagePerReactivePowerControlAdderImpl.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm.network.extensions;
+
+import com.powsybl.commons.extensions.AbstractExtensionAdder;
+import com.powsybl.iidm.network.StaticVarCompensator;
+
+/**
+ * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
+ */
+public class VoltagePerReactivePowerControlAdderImpl
+        extends AbstractExtensionAdder<StaticVarCompensator, VoltagePerReactivePowerControl>
+        implements VoltagePerReactivePowerControlAdder {
+
+    private double slope;
+
+    protected VoltagePerReactivePowerControlAdderImpl(StaticVarCompensator svc) {
+        super(svc);
+    }
+
+    @Override
+    protected VoltagePerReactivePowerControlImpl createExtension(StaticVarCompensator svc) {
+        return new VoltagePerReactivePowerControlImpl(svc, slope);
+    }
```

Comment:
There is something strange in the design: it's not related to your PR, but why the adder take one `Extendable` instance and the createExtension could take another one?
@MioRtia : what do you think?

Reply:
Hm, good question. In the code, `createExtension` is only called with the `extendable` field (from the constructor) as parameter. Maybe the method should not take any parameter and `extendable` should be protected? (@jonenst do you remember why you designed it this way?)

Reply:
I did it this way because it was more explicit for an implementer of an adder to take the extendable as a method argument from the method he is supposed to implement. The alternative (that you mentionned) is to have the extendable in a protected field of the base class and document that it should be used in the createExtension method

Reply:
I'm ok with both implementations if you want to change it

Reply:
I guess it's not so implicit with a protected field because you have to write the constructor calling super() with the extendable so you are hinted about it

Reply:
Ok, thank you for the explanation! Both are okay for me, what do you think @mathbagu ? In any case, I think this PR is good to be merged, we can make another PR on this subject later.

Reply:
The only thing I want we ensure, is that should be forbidden to pass a different instance. I think it could be the case with the abstract class we provide. It's not directly related to this PR, so I will merge it but the discussion is still open